### DISCUSSION TITLE
Multiple GPU Utilization

### DISCUSSION THREAD

**Discussion by ben-barnard** [neonbjb/tortoise-tts#75] [Q&A] [answered]
Hello! I've been having a blast with your project and really learning a lot. Quick question (and apologies if it's obvious) - can this code take advantage of multiple GPU setups? I have a couple 1070tis sitting idle. Thank you!

**Comment by neonbjb** [comment, marked as answer]
You can run multiple scripts in parallel by setting the CUDA_VISIBLE_DEVICES environment variable, so for example `CUDA_VISIBLE_DEVICES=1 python do_tts.py` will run do_tts on the 2nd GPU.

**Reply by ben-barnard** [reply]
Terrific, thank you!
